tests/unit/s2n_rc4_test.c

```diff
@@ -33,6 +33,7 @@
 
 int main(int argc, char **argv)
 {
+#ifdef LIBCRYPTO_SUPPORTS_EVP_RC4
```

Comment:
By #ifdef'ing out the whole test, we don't print the standard message about the test being skipped. Can you put the #ifdef between the BEGIN_TEST and END_TEST macros?